### PR TITLE
2-way ANOVA support

### PR DESCRIPTION
**Pull request by hideaki**
# Description
2-way ANOVA support

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/test_wrapper.R [hunk 1]
```diff
@@ -1347,20 +1347,32 @@ exp_anova <- function(df, var1, var2, covariates = NULL, func2 = NULL, covariate
     power <- 1.0 - beta
   }
   var1_col <- col_name(substitute(var1))
-  var2_col <- col_name(substitute(var2))
+  var2_ <- substitute(var2)
```
Comment (kei51e) [inline R/test_wrapper.R:1350]: Add a comment for var1 and var2 at the function declaration.

File: R/test_wrapper.R [hunk 2]
```diff
@@ -1761,19 +1863,19 @@ tidy.anova_exploratory <- function(x, type="model", conf_level=0.95, levene_test
     orig_terms[is.na(orig_terms)] <- colnames(ret)[is.na(orig_terms)] # Fill the column names that did not have a matching mapping.
     colnames(ret) <- orig_terms
   }
-  else if (type == "prob_dist") {
+  else if (type == "prob_dist") { # Note that 2-way ANOVA, with multiple P-values in the result, is not supported here.
```
Comment (kei51e) [inline R/test_wrapper.R:1866]: Update comment.

File: R/test_wrapper.R (hunk 1 above)
Comment (hideaki) [reply R/test_wrapper.R:1350]: Done.

File: R/test_wrapper.R [hunk 3]
```diff
@@ -1391,11 +1407,17 @@ exp_anova <- function(df, var1, var2, covariates = NULL, func2 = NULL, covariate
         covariates <- name_map[covariates]
       }
       df <- clean_df
-
-      if (is.null(covariates)) {
-        formula <- as.formula(paste0('`', var1_col, '`~`', var2_col, '`'))
+      if (is.null(covariates)) { # 2-way/1-way ANOVA case
+        if (with_interaction) {
+          # TODO: With *, car::Anova(x, type="III") fails with "there are aliased coefficients in the model" when there are empty cells.
```
[the remark this answers is not in the document]
Comment (hideaki) [reply R/test_wrapper.R:1412]: Done.

File: R/test_wrapper.R (hunk 2 above)
Comment (hideaki) [reply R/test_wrapper.R:1866]: Done (Removed the comment that does not hold true anymore.)

File: tests/testthat/test_test_wrapper_1.R [hunk 4]
```diff
@@ -605,17 +604,53 @@ test_that("test exp_ttest with group-level error (not eough data)", {
   expect_equal(nrow(ret), 0)
 })
 
+test_that("test 2-way ANOVA with exp_anova", {
```
[the remark this answers is not in the document]
Comment (hideaki) [reply tests/testthat/test_test_wrapper_1.R:607]: Added the test.

**Review by kei51e — APPROVED** [review]

**Pull request merged**